docs/components.md

```diff
@@ -57,3 +57,7 @@ Models must be compiled and added to `window.kiln.componentModels` for Kiln to c
 Schemas \(or _schemata_, for pedants like myself\) are the main connective tissue between components and Kiln. They define what kinds of fields components have, as well as how those fields are grouped. They also provide `_description` text for the components' Info Modals, `_version` for upgrading data programmatically, and `_confirmRemoval` for extra peace-of-mind on components like ads.
 
 Schemas for components that are currently on the page are automatically parsed by Amphora and added to the page when Kiln loads. Schemas for other components are added when those components are added.
+
+## Variations
+
+Components can have variations, which is defined by corresponding stylesheets in the site's styleguide folder. There is the default variation (`componet-name.css`) and variations are defined as `component-name_variation.css`. Kiln will automatically generate a select input with the variations and a 'Component Variation' tab in the Settings form. If the component doesn't have Settings, Kiln will generate this too.
```

Comment:
typo `(`componet-name.css`)`. Also not sure what you mean by this: `If the component doesn't have Settings, Kiln will generate this too.`